Tests/UnitTests/Core/Utilities/ContextTypeTests.cpp

```diff
@@ -0,0 +1,38 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include <boost/test/tools/old/interface.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Utilities/detail/ContextType.hpp"
+
+using namespace Acts;
+
+BOOST_AUTO_TEST_SUITE(ContextTypeTests)
+
+BOOST_AUTO_TEST_CASE(PackUnpack) {
+  ContextType ctx;
+
+  int v = 42;
+  ctx = v;
```

Comment:
```suggestion
  ctx = int{42};
```

I think we allow uniform initialization.

Reply:
Clang tidy had complained about `int(42)`, but possibly `int{42}` would work. Ultimately, I think this is fine now.

Reply:
I think we leave it like that, it's a test after all.